From 70a0bcde87512c269269443444c109740dcacc19 Mon Sep 17 00:00:00 2001
From: Chen Ni <nichen@iscas.ac.cn>
Date: Wed, 24 Sep 2025 10:05:40 +0800
Subject: [PATCH] ASoc: tas2783A: Remove unneeded semicolon
Git-commit: 70a0bcde87512c269269443444c109740dcacc19
Patch-mainline: v6.18-rc1
References: jsc#PED-14430 jsc#PED-14297 jsc#PED-14024

Remove unnecessary semicolons reported by Coccinelle/coccicheck and the
semantic patch at scripts/coccinelle/misc/semicolon.cocci.

Signed-off-by: Chen Ni <nichen@iscas.ac.cn>
Link: https://patch.msgid.link/20250924020540.234560-1-nichen@iscas.ac.cn
Signed-off-by: Mark Brown <broonie@kernel.org>
Acked-by: Takashi Iwai <tiwai@suse.de>

---
 sound/soc/codecs/tas2783-sdw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sound/soc/codecs/tas2783-sdw.c b/sound/soc/codecs/tas2783-sdw.c
index 3e03e68932f6..d5d33afdfa10 100644
--- a/sound/soc/codecs/tas2783-sdw.c
+++ b/sound/soc/codecs/tas2783-sdw.c
@@ -806,7 +806,7 @@ static void tas2783_fw_ready(const struct firmware *fmw, void *context)
 			offset += file_blk_size;
 		else
 			break;
-	};
+	}
 	mutex_unlock(&tas_dev->pde_lock);
 	tas2783_update_calibdata(tas_dev);
 
-- 
2.52.0

